test/util/regtest.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+const assert = require('./assert');
+const FullNode = require('../../lib/node/fullnode');
+const SPVNode = require('../../lib/node/spvnode');
+
+const {NodeClient, WalletClient} = require('bclient');
+
+async function initFullNode(options) {
+  const node = new FullNode({
+    prefix: options.prefix,
+    network: 'regtest',
+    apiKey: 'foo',
+    walletAuth: true,
+    workers: true,
+    listen: true,
+    bip37: true,
+    port: options.ports.full.p2p,
+    httpPort: options.ports.full.node,
+    maxOutbound: 1,
+    seeds: [],
+    memory: false,
```

Comment:
Using this in bPanel tests, I think it would be useful to support passing `memory: true` in options which makes cleanup easier.

Reply:
Cleanup is already simple as it's stored in `/tmp` and cleaned up before tests run.

Reply:
Also, I want to make sure that the tests are running with LevelDB.

Reply:
Having `memory` be part of the `options`, should be fine though.

Reply:
Actually, having a memory option here could be a bit misleading as there is the `prefix` option already for the data directory. There also isn't any code here that would exercise it yet.

In a side note: The use case for using memory is exclusively in tests, and could be something that is stubbed or mocked rather than being a configuration option for a node. I think it's also set as true by default, which has already be the source of a bug where the SPV node was accidently configured to be in memory, see https://github.com/bcoin-org/bcoin/pull/456.